recipe/meta.yaml

```
@@ -118,7 +114,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('ray-core', exact=True) }}
-        - aiohttp
+        - aiohttp =3.7
```

Yeah, unless 3.7 is absolutely required as a lower bound, I agree with @vnlitvinov that this should be:
```suggestion
        - aiohttp <3.8
```